whelk-core/src/main/groovy/whelk/util/Unicode.groovy

```diff
@@ -0,0 +1,32 @@
+package whelk.util
+
+import java.text.Normalizer
+
+class Unicode {
+    private static final List FORBIDDEN_UNICODE_CHARS = [
+            'ﬀ', // 'LATIN SMALL LIGATURE FF'
+            'ﬃ', // 'LATIN SMALL LIGATURE FFI'
+            'ﬄ', // 'LATIN SMALL LIGATURE FFL'
+            'ﬁ', // 'LATIN SMALL LIGATURE FI'
+            'ﬂ', // 'LATIN SMALL LIGATURE FL'
+            'ﬅ', // 'LATIN SMALL LIGATURE LONG S T'
+            'ﬆ', // 'LATIN SMALL LIGATURE ST'
```

Comment:
Are these cherry-picked or from a standardized set (e.g. of typographic ligatures)? Can we reference something, e.g. https://en.wikipedia.org/wiki/Orthographic_ligature or similar as a "source"?)